README.md

```diff
@@ -444,14 +437,10 @@ Or set the following environment variable before importing JAX:
 XLA_FLAGS=--xla_gpu_cuda_data_dir=/path/to/cuda
 ```
 
-The Python version must match your Python interpreter. There are prebuilt wheels
-for Python 3.6, 3.7, and 3.8; for anything else, you must build from source. Jax
-requires Python 3.6 or above. Jax does not support Python 2 any more.
-
 To try automatic detection of the correct version for your system, you can run:
 
 ```bash
-pip install --upgrade https://storage.googleapis.com/jax-releases/`nvcc -V | sed -En "s/.* release ([0-9]*)\.([0-9]*),.*/cuda\1\2/p"`/jaxlib-0.1.52-`python3 -V | sed -En "s/Python ([0-9]*)\.([0-9]*).*/cp\1\2/p"`-none-manylinux2010_x86_64.whl jax
+pip install --upgrade jax jaxlib==0.1.52+cuda`nvcc -V | sed -En "s/.* release ([0-9]*)\.([0-9]*),.*/\1\2/p"` -f https://storage.googleapis.com/jax-releases/jax_releases.html
```

Comment:
I think I'd just write this in two lines:

* use `nvcc --version` to find out your CUDA versoin
* then run `...`

Less cryptic and just as easy to understand I think.

Reply:
We basically already have that above. I just removed the auto install, presumably it's now easy enough to find your cuda version and where to put it in the command, like you say.

Reply:
But my point is I'd just remove this cryptic one-liner. I think it hurts more than it helps.

Reply:
It's already removed :)